rest-server/src/models/job.js

```diff
@@ -404,6 +410,32 @@ class Job {
     return frameworkDescription;
   }
 
+  _initializeJobContextRootFolders(next) {
+    const hdfs = new Hdfs(launcherConfig.webhdfsUri);
+    async.parallel([
+      (parallelCallback) => {
+        hdfs.createFolder(
+          '/Output',
+          {'user.name': 'root', 'permission': '777'},
```

Comment:
what if the username is omitted, can we use the default username and permission?

Reply:
Default username is 'dr.who' and default permission is 755 (please see the [API doc](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/WebHDFS.html#Make_a_Directory)) -- other users won't have permission to create sub-folders.

Reply:
I mean `/Container` dir where user do not need to access could have permission 755.
As for default user, native hdfs lib will use `root`.